src/pages/iou/request/step/IOURequestStepScan/index.tsx

```diff
@@ -862,6 +865,18 @@ function IOURequestStepScan({
         </>
     );
 
+    const elementTopOffset = useMemo(() => {
+        if (platform === CONST.PLATFORM.MOBILEWEB) {
+            return 10;
+        }
+
+        if (isSmallScreenWidth) {
+            return -variables.tabSelectorButtonHeight + 10;
+        }
+
+        return -variables.tabSelectorButtonHeight + 8;
+    }, [isSmallScreenWidth, platform]);
+
```

Comment:
could we add these hardcoded ints to variables as well?

Reply:
Done!